components/portal/messages/partials/notifications-list-item.html

```diff
@@ -19,23 +19,35 @@
 
 -->
 <!-- NOTIFICATION CONTENT -->
-<a class="md-list-item-text"
+<div class="md-list-item-text"
    ng-href="{{ notification.actionButton.url }}"
-   target="_blank"
-   rel="noopener noreferrer"
-   aria-label="{{ notification.actionButton.label }}"
    layout="row"
    layout-align="start center"
    layout-xs="column"
    layout-align-xs="center start">
-  <span><md-icon ng-if="notification.priority == 'high'" aria-label="high priority:">priority_high</md-icon> {{ notification.title }}</span>
-  <div ng-if="notification.moreInfoButton" class="notification-buttons" layout-xs="column" layout-align-xs="center start">
+  <!-- Notification text -->
+  <div layout="column" layout-align="center start" flex-gt-xs="70">
+    <div class="notification__priority-message" ng-if="notification.priority === 'high'">
+      <md-icon class="md-warn" aria-label="high priority:">priority_high</md-icon>
```

Comment:
Is this `aria-label` redundant with the span following?

Reply:
Yes. Yes it is.